tests/units/modules/clustering/kubevirt/test_kubevirt_raw.py

```diff
@@ -0,0 +1,66 @@
+import json
+import sys
+import pytest
+
+from ansible.compat.tests.mock import patch, MagicMock
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+
+from openshift.dynamic import Resource
+
+# FIXME: paths/imports should be fixed before submitting a PR to Ansible
+sys.path.append('lib/ansible/modules/clustering/kubevirt')
+
+import kubevirt_raw as mymodule
+
+
+def set_module_args(args):
+    args = json.dumps({'ANSIBLE_MODULE_ARGS': args})
+    basic._ANSIBLE_ARGS = to_bytes(args)
+
+
+class AnsibleExitJson(Exception):
```

Comment:
Why not to use `exceptions.NotFoundError` from the dynamic client.

Reply:
Dunno. I'll make note to take a look into it after I've written everything.

Reply:
It is much better to look for errors which can actually occur in the code.

Reply:
I've looked into it. That exception is not an error, it's a method of injecting into the underlying AnsibleModule base class an exit point for our code to take back control. It's the correct option to explicitly define an arbitrary exception in this case – using anything else would falsely imply there is some other meaning behind it other than "we need the underlying module to stop executing and give us back control at this precise point". Which there isn't.

Reply:
OK